src/main/resources/swagger/api-docs.yaml

```diff
@@ -2351,6 +2351,29 @@ paths:
         500:
           description: Internal Error
 
+  /api/staticNotebooks/convert:
+    post:
+      x-passthrough: true
+      x-passthrough-target: calhoun
```

Comment:
The repo was renamed to `notebook-preview`, should this match? https://github.com/DataBiosphere/notebook-preview

Reply:
The code was actually pulled *back* to Calhoun more recently: https://github.com/DataBiosphere/calhoun/pull/7

Reply:
Huh, thanks - looks good then.

Reply:
This was done to preserve code history in one place. We could still consider renaming calhoun to something more descriptive.